library/src/main/res/values/strings.xml

```diff
@@ -24,7 +24,11 @@
     <string name="stream_input_upload_media">Upload a photo or video</string>
     <string name="stream_input_take_picture">Upload from a camera</string>
     <string name="stream_input_upload_file">Upload a file</string>
-
+    <!-- Folder Name -->
+    <string name="stream_image">Images</string>
```

Comment:
The name is too generic. Let's call it clearly `stream_dir_image`. As well as others.